conu/docker/core.py

```diff
@@ -147,3 +148,22 @@ def copy_to(self, src, dest):
     def copy_from(self, src, dest):
         self.start()
         run_cmd("docker cp %s:%s %s" % (self.tag, src, dest))
+
+    def check_files_exist(self, files):
+        """
+        Function returns
+        :param files:
+        :return: True if all files exists
+                 False if at least one does not exist
+        """
+        found = True
+        if isinstance(files, str):
+            files = [files]
+        if files:
+            for f in files:
+                logger.debug("Check if %s file is present in container." % f)
+                ret_val = self.execute('ls %s' % f)
+                if f != ret_val.strip():
+                    found = False
```

Comment:
does it make sense to continue since the log below does not output the name of the file?